packages/http-client-python/emitter/src/emitter.ts

```diff
@@ -282,3 +284,54 @@ async function setupPyodideCall(root: string) {
   }
   return pyodide;
 }
+
+async function runBlack(venvPath: string, outputDir: string): Promise<void> {
+  const blackCommand = `${venvPath} -m black --line-length=120 --fast ${outputDir}`;
```

Comment:
There may be many unexpected folders under outputdir which doesn't need black, thus it is better to filter those paths so that the speed could be faster:
https://github.com/microsoft/typespec/blob/376d426742477dec51bbaad4376120a3298de641/packages/http-client-python/generator/pygen/black.py#L40-L45

Reply:
Done!